deepchem/feat/base_classes.py

```diff
@@ -50,9 +50,8 @@ def featurize(self, datapoints: Iterable[Any],
         features.append(self._featurize(point))
       except:
         logger.warning(
-            "Failed to featurize datapoint %d. Appending empty array")
+            "Failed to featurize datapoint %i. Appending empty array" % i)
```

Comment:
This should be %d for integer input in the string.

Reply:
["For output, %i and %d are the exact same thing, both in Python and in C."](https://stackoverflow.com/questions/17680256/what-is-the-difference-between-i-and-d-in-python)

I used %i to match the %i in line 48. Would you like me to change that?